### PR TITLE
feat(ignore): add option to pass pkgs to be ignored

### PR DESCRIPTION
**Pull request by ruxandrafed**
An MVP for #2.

I could see this getting more complex, where we could have a config file and pass more complex exceptions, but this should be fine for now.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -81,12 +81,13 @@ OUTDATED        ^3.5.1 → 6.4.1          npm @ ^3.5.1
...
-| `--silent` | no putput on `stderr`                    |
-| `--color`  | pretty colors!                           |
-| `--update` | update `package.json` to latest versions |
+| Name       | Description                                            |
+| ---------- | -------------------------------------------------------|
+| `--json`   | output JSON results to `stdout`                        |
+| `--silent` | no output on `stderr`                                  |
+| `--color`  | pretty colors!                                         |
+| `--update` | update `package.json` to latest versions               |
+| `--ignore` | pass package names to be ignored, e.g. `--ignore=tap`  |
```
Comment (ahmadnassri) [inline README.md]: description missing comma separation?

File: index.js [hunk 2]
```diff
@@ -2,12 +2,19 @@
 
 const { writeFileSync } = require('fs')
 
+const getIgnoreList = () => {
+  const arg = process.argv.find(arg => arg.match('--ignore='))
+  if (!arg) return []
+  return arg.match(/--ignore=(.*)$/)[1].split(',')
```
Comment (ahmadnassri) [inline index.js]: use `.shift()` or `.pop()` here instead of referencing the direct array index?

Comment (ruxandrafed) [reply index.js]: How about this? I think it's more readable than using `match` again anyway.

https://github.com/ahmadnassri/node-updated/blob/573c30bfe0883365e24a83676a3e4fa7f499be86/index.js#L8

**Pull request merged**